Formula/pybind11.rb

```diff
@@ -44,8 +57,17 @@ def install
       example.add(1,2)
     EOS
 
+    version = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    site_packages = "lib/python#{version}/site-packages"
+
     python_flags = `#{Formula["python@3.9"].opt_bin}/python3-config --cflags --ldflags --embed`.split
     system ENV.cxx, "-O3", "-shared", "-std=c++11", *python_flags, "example.cpp", "-o", "example.so"
     system Formula["python@3.9"].opt_bin/"python3", "example.py"
+
+    test_module = shell_output("#{Formula['python@3.9'].opt_bin/'python3'} -m pybind11 --includes")
```

Comment:
```suggestion
    test_module = shell_output("#{Formula["python@3.9"].opt_bin/"python3"} -m pybind11 --includes")
```
This is inside an interpolation so you're safe to use `"` here. Same below